index.js

```diff
@@ -0,0 +1,22 @@
+
+
+document.addEventListener("DOMContentLoaded", () =>{
+    const fecthAllCards = async () => {
+       let newDeck = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1")
+       let deckId = newDeck.data.deck_id
```

Comment:
Have the deckId outside this function scope since you're going to need the id after the function runs.